src/bin/tikv-server.rs

```diff
@@ -381,192 +382,206 @@ fn get_rocksdb_db_option(config: &toml::Value) -> RocksdbOptions {
-    opt
+    opts
 }
 
 fn get_rocksdb_lock_cf_option(config: &toml::Value) -> RocksdbOptions {
-    let mut opts = RocksdbOptions::new();
-    let mut block_base_opts = BlockBasedOptions::new();
-    block_base_opts.set_block_size(16 * 1024);
-
-    let block_cache_size = get_toml_int(config,
-                                        "rocksdb.lockcf.block-cache-size",
-                                        Some(256 * 1024 * 1024));
-    block_base_opts.set_lru_cache(block_cache_size as usize);
-
-    block_base_opts.set_bloom_filter(10, false);
-    opts.set_block_based_table_factory(&block_base_opts);
-
-    let cpl = "no:no:no:no:no:no:no".to_owned();
-    let per_level_compression = util::config::parse_rocksdb_per_level_compression(&cpl)
-        .unwrap_or_else(|err| exit_with_err(format!("{:?}", err)));
-    opts.compression_per_level(&per_level_compression);
-
-    let write_buffer_size = get_toml_int(config,
-                                         "rocksdb.lockcf.write-buffer-size",
-                                         Some(128 * 1024 * 1024));
-    opts.set_write_buffer_size(write_buffer_size as u64);
-
-    let max_write_buffer_number =
-        get_toml_int(config, "rocksdb.lockcf.max-write-buffer-number", Some(5));
-    opts.set_max_write_buffer_number(max_write_buffer_number as i32);
-
-    let max_bytes_for_level_base = get_toml_int(config,
-                                                "rocksdb.lockcf.max-bytes-for-level-base",
-                                                Some(128 * 1024 * 1024));
-    opts.set_max_bytes_for_level_base(max_bytes_for_level_base as u64);
-    opts.set_target_file_size_base(32 * 1024 * 1024);
-
-    // set level0_file_num_compaction_trigger = 1 is very important,
-    // this will result in fewer sst files in lock cf.
-    opts.set_level_zero_file_num_compaction_trigger(1);
-
-    let level_zero_slowdown_writes_trigger = get_toml_int(config,
-                                                          "rocksdb.lockcf.\
-                                                           level0-slowdown-writes-trigger",
-                                                          Some(20));
-    opts.set_level_zero_slowdown_writes_trigger(level_zero_slowdown_writes_trigger as i32);
-
-    let level_zero_stop_writes_trigger = get_toml_int(config,
-                                                      "rocksdb.lockcf.level0-stop-writes-trigger",
-                                                      Some(36));
-    opts.set_level_zero_stop_writes_trigger(level_zero_stop_writes_trigger as i32);
-
-    opts
+    let mut default_values = CfOptValues::default();
+    default_values.block_size = 16 * KB as i64;
+    default_values.use_bloom_filter = true;
+    default_values.whole_key_filtering = true;
+    default_values.compression_per_level = String::from("no:no:no:no:no:no:no");
+    default_values.level_zero_file_num_compaction_trigger = 1;
```

Comment:
if the write buffer is 128MB, level_zero_file_num_compaction_trigger is 1, the L0 will be 128MB, so L1 should be 128MB as RocksDB suggests, but the max_bytes_for_level_base is 512MB here, is it ok?